_schema.graphql

```diff
@@ -8040,14 +8065,18 @@ type Money {
   major: Float!
 }
 
+# A simple type to hold an amount and currency code
 input MoneyInput {
-  # amount in dollars or major unit
-  amount: Float!
+  # Amount in the minor unit of currency
+  amountMinor: Int!
```

Comment:
This schema doesn't match the[ v1 field definition of `MoneyInput`](https://github.com/artsy/metaphysics/blob/f885199a5ababedcd0b81fdbd55395dc48f9a98c/src/schema/v1/fields/money.ts#L146), and that's causing some v1 tests to fail for BNMO mutations.

Depending on why this change was made, I think that we will either need to fix the schema or the field definition to make them match.